src/Raven.Server/Config/Categories/SubscriptionConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using Raven.Server.Config.Attributes;
+using Raven.Server.Config.Settings;
+
+namespace Raven.Server.Config.Categories
+{
+    public class SubscriptionConfiguration : ConfigurationCategory
+    {
+        [DefaultValue(10)]
```

Comment:
How many subscriptions did you run on your machine before it became a big issue?